File: program/find-the-average-of-all-numbers-in-an-array/FindTheAverageOfAllNumbersInAnArray.c
#include <stdio.h>

int main() {
    int n, i;
    float sum = 0.0, avg;

    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter the elements of the array:\n");

    for(i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        sum += arr[i];
    }

    avg = sum / n;

    printf("The average of the elements in the array is: %f", avg);
    
    return 0;
}

File: program/find-the-average-of-all-numbers-in-an-array/find_the_average_of_all_numbers_in_an_array.dart
void avgOfArray(List<int> arr){
  double avg = 0;
  for(var i = 0 ; i < arr.length ; i++){
    avg += arr[i];
  }
  avg = avg / arr.length;
  print(avg);
}

void main() {
  avgOfArray([1,2,3,6,7]);
}